tests/cl-genesis-positions/convert.go

```diff
@@ -186,8 +186,8 @@ func ConvertSubgraphToOsmosisGenesis(positionCreatorAddresses []sdk.AccAddress,
 		position, err := clMsgServer.CreatePosition(sdk.WrapSDKContext(osmosis.Ctx), &cltypes.MsgCreatePosition{
 			PoolId:          poolId,
 			Sender:          randomCreator.String(),
-			LowerTick:       lowerTickOsmosis.Int64(),
-			UpperTick:       upperTickOsmosis.Int64(),
```

Comment:
oh wow the positions were always in int64's

Reply:
Yeah I think we've refactored this multiple times due to changes to tick spacing model from the original so it might not be very consistent everywhere